workload/aspnetapp.yaml

```diff
@@ -46,8 +46,8 @@ spec:
                   - traefik-ingress-ilb
               topologyKey: "kubernetes.io/hostname"
       containers:
-      - name: aspnetcore-webapp-sample
-        image: mcr.microsoft.com/dotnet/core/samples:aspnetapp
+      - name: aspnet-webapp-sample
+        image: mcr.microsoft.com/dotnet/samples:aspnetapp
         imagePullPolicy: Always
         securityContext:
           allowPrivilegeEscalation: false
```

Comment:
Not this line, but the line below (GH won't let me target it), I'm pretty sure is UNSUPPORTED for aspnet app files. Disk access is required for paging.  I know it's not in scope, but technically this change wasn't either. I think you can set this to `false` and then remove the `COMPlus_EnableDiagnostics` "workaround" below with the env variable.  I don't think that's the best practice for these base images.

Reply:
I need to rebase, because the new Net5 app is already in master.
About your comment I need to investigate. Good point.